### DISCUSSION TITLE
[Script request] Duplicati

### DISCUSSION THREAD

**Discussion by Hjord** [community-scripts/ProxmoxVE#126] [Request script]
### Application Name

Duplicati

### Website

https://github.com/duplicati/duplicati

### Description

Duplicati is a free, open source, backup client that securely stores encrypted, incremental, compressed backups on cloud storage services and remote file servers. It works with:

   Amazon S3, [IDrive e2](https://www.idrive.com/e2/duplicati), [Backblaze (B2)](https://www.backblaze.com/blog/duplicati-backups-cloud-storage/), Box, Dropbox, FTP, Google Cloud and Drive, MEGA, Microsoft Azure and OneDrive, Rackspace Cloud Files, OpenStack Storage (Swift), Sia, Storj DCS, SSH (SFTP), WebDAV, Tencent Cloud Object Storage (COS), Aliyun OSS, [and more!](https://duplicati.readthedocs.io/en/latest/01-introduction/#supported-backends)

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
merged
